src/Microsoft.PowerShell.Commands.Management/commands/management/Process.cs

```diff
@@ -189,13 +189,20 @@ private void RetrieveMatchingProcessesByProcessName()
                 if (!found &&
                     !WildcardPattern.ContainsWildcardCharacters(pattern))
                 {
+                    string val = "NoProcessFoundForGivenName";
+                    string errorText = ProcessResources.NoProcessFoundForGivenName;
+                    if(int.TryParse(pattern, out int x) && x>0)
```

Comment:
nit: add some whitespace around the `>`
```suggestion
                    if(int.TryParse(pattern, out int x) && x > 0)
```